openlibrary/core/lending.py

```diff
@@ -157,13 +157,18 @@ def compose_ia_url(limit=None, page=1, subject=None, query=None, work_id=None,
                         authors.append(author_name)
                         authors.append(','.join(author_name.split(' ', 1)[::-1]))
                     if authors:
-                        _q = ' OR '.join(['creator:"%s"' % author for author in authors])
+                        _q = ' OR '.join('creator:"%s"' % author for author in authors)
```

Comment:
IMHO this change isn't topical to this pull request, but it's also a good change, so, cool :)